src/classNames.js

```diff
@@ -0,0 +1,15 @@
+import style from 'flexboxgrid';
+
+function defaultClassNameLookup(className) {
+  return style[className] || className;
```

Comment:
In case CSS Modules aren't enabled, `style` will be `undefined` or something like that, no? In that case `style[className]` would throw, so I suggest doing this:

```js
return style ? style[className] : className;
```

Unless I'm mistaken. In that case please correct me.